doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -7,22 +7,173 @@ IronSide Secure Element
+
+By default, the nRF54H20 SoC uses the following memory and access configurations:
+
+* MRAMC configuration: MRAM operates in Direct Write mode.
+* MPC configuration: All memory not reserved by Nordic firmware is accessible with read, write, and execute (RWX) permissions by any domain.
+* TAMPC configuration: The access port (AP) for the application core is enabled and available, allowing direct programming of all the memory not reserved by Nordic firmware in the default configuration.
+
+.. note::
+   Access to external memory (EXMIF) requires a non-default configuration of the GPIO.CTRLSEL register.
+
+Global domain memory can be protected from write operations by configuring UICR registers.
+To remove these protections and disable all other protection mechanisms enforced through UICR settings, perform an ``ERASEALL`` operation.
+
+.. _ug_nrf54h20_ironside_se_update:
+
+Updating |ISE|
+**************
+
+|NCS| supports two methods for updating the |ISE| firmware on the nRF54H20 SoC:
+
+* Using the ``west`` command.
+  You can use the ``west`` command provided by the |NCS| to install the firmware update.
+  For step-by-step instructions, see :ref:`ug_nrf54h20_ironside_se_update_west`.
+
+* Updating the SoC binaries manually.
+  Alternatively, you can perform the update by manually executing the same steps carried out by the ``west`` command.
+  For step-by-step instructions, see :ref:`ug_nrf54h20_ironside_se_update_manual`.
+
+.. caution::
+   You cannot update the nRF54H20 SoC binaries from a SUIT-based (up to 0.9.6) to an IronSide-SE-based (2x.x.x) version.
+
+.. _ug_nrf54h20_ironside_se_update_west:
+
+Updating using west
+===================
+
+To update the |ISE| firmware, you can use the ``west ncs-ironside-se-update`` command with the following syntax:
+
+.. code-block:: console
+
+   west ncs-ironside-se-update --zip <path_to_soc_binaries.zip> --allow-erase
+
+The command accepts the following main options:
+
+* ``--zip`` (required) - Sets the path to the nRF54H20 SoC binaries ZIP file.
+* ``--allow-erase`` (required) - Enables erasing the device during the update process.
+* ``--serial`` - Specifies the serial number of the target device.
+* ``--firmware-slot`` - Updates only a specific firmware slot (``uslot`` for |ISE| or ``rslot`` for |ISE| Recovery).
+* ``--wait-time`` - Specifies the timeout in seconds to wait for the device to boot (default: 2.0 seconds).
+
+.. _ug_nrf54h20_ironside_se_update_manual:
+
+Updating manually
+=================
+
+The manual update process involves the following steps:
+
+1. Executing the update application.
+   The update application runs on the application core and communicates with the |ISE| update service.
+   It reads the update firmware from memory and passes the update blob metadata to the |ISE|.
+
+#. Preparing the update.
+   The |ISE| validates the update parameters and writes the update metadata to the Secure Information Configuration Registers (SICR).
```

Comment:
Should we describe that the local domain requests this through the service?

Currently it seems like IronSide does this implicitly.

Reply:
I merged "Executing the update application." and "Preparing the update."

And now I think it more obvious that this happens via the update service.

(not pushed yet)